src/libraries/System.Text.Json/tests/Common/UnsupportedTypesTests.cs

```diff
@@ -129,8 +129,6 @@ public static IEnumerable<object[]> GetUnsupportedValues()
             yield return WrapArgs((IntPtr)123);
             yield return WrapArgs<IntPtr?>(new IntPtr(123)); // One nullable variation.
             yield return WrapArgs((UIntPtr)123);
-            yield return WrapArgs((Memory<byte>)new byte[] { 1, 2, 3 });
```

Comment:
You might also want to remove a few `[JsonSerializable]` declarations on the `JsonSerializerContext` defined for these tests.